### ISSUE TITLE
Rendu BANO très en retard ? très lent ?

### ISSUE THREAD

**Issue by Cyrille37** [osm-fr/infrastructure#277]
Bonjour
J'ai l'impression qu'en ce moment le rendu BANO n'est pas à jour. J'ai saisi des rues il y a 4 jours et elles sont toujours en rouge sur le rendu (ex [Route de Narbonne, Joué les Tours](https://www.openstreetmap.org/way/70606740/history)).

Aussi le rendu est très très pixelisé, souvent impossible à lire même après 1 heure d'attente.

Bon courage et merci pour ces supers service :-)

**Comment by Cyrille37** [comment]
Peut être lié à #276 ?

**Comment by cquest** [comment]
Oui, le rendu BANO est fait par osm13, comme tous les rendus "techniques"

**Comment by cquest** [comment]
Retour à la normale... depuis le retour d'osm13

**Comment by Cyrille37** [comment]
Un grand merci <3 !!!
